CONTRIBUTING.md

```diff
@@ -245,13 +251,27 @@ If in doubt, you can always ask for guidance in the Pull Request or on
 Feel free to post a comment in the Pull Request to ping reviewers if you are
 awaiting an answer on something.
 
+Notes: if the reviewer mentions *nits* in their comments, that stands for
```

Comment:
I don't think "stands for" is the right wording. That makes it sound like it's an acronym, which it isn't.